### ISSUE TITLE
Alt text is not descriptive [../interactions files]

### ISSUE THREAD

**Issue by timngyn** [aws-amplify/docs#7067]
**Describe the content issue**:
Images listed below do not have descriptive/informative alt text. For markdown the alt text is located between the square brackets, `![alt text](...)`.

The list below isn't exhaustive since I found these images based on the word count for the alt text, but it should be a good starting point to make our alt text better for people who use screen readers.

Some resources to help write alt text:
- https://webaim.org/techniques/alttext/
- https://www.a11yproject.com/posts/are-you-making-these-five-mistakes-when-writing-alt-text/

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib/interactions/js/getting-started.mdx:
```
   16  
   17: ![Interactions](/images/interactions_lex_v2_console_edit_bot.png)
   18  

  104  
  105: ![Interactions](/images/interactions_lex_console_edit_bot.jpg)
  106  
```

&nbsp;

https://github.com/aws-amplify/docs/blob/main/src/fragments/lib-v1/interactions/js/getting-started.mdx:
```
  49  
  50: ![Interactions](/images/interactions_lex_console_edit_bot.jpg)
  51  

  77  
  78: ![Interactions](/images/interactions_lex_v2_console_edit_bot.png)
  79  
```
